MIT.md

```diff
@@ -1,4 +1,4 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [Yinka Olatunji]
```

Comment:
- [ ] Please consider Updating your `README` file by customizing it to be more specific for this project, kindly add your project name to the readme file as well as filling out the other sections such as the `getting started` describing how would other people could use this repo.